integration/src/test/kotlin/kotlinx/benchmark/integration/InvalidTargetingTest.kt

```diff
@@ -0,0 +1,50 @@
+package kotlinx.benchmark.integration
+
+import kotlin.test.*
+
+class InvalidTargetingTest : GradleTest() {
+
+    @Test
+    fun testInvalidTargetForKotlinWASM() {
+        val runner = project("kotlin-multiplatform") {
+            configuration("invalidWasm") {
+                iterations = 1
+                iterationTime = 100
+                iterationTimeUnit = "ms"
+                reportFormat = "json"
+            }
+            register("wasm")
+            kotlin {
+                wasm("wasm") {
+                    nodejs()
+                }
+            }
+        }              
+
+        runner.runAndFail("invalidWasmBenchmark") {
+            assertOutputContains("Kotlin/WASM does not support targeting NodeJS for benchmarks.")
+        }
+    }
+
+    @Test
+    fun testInvalidTargetForKotlinJS() {
+        val runner = project("kotlin-multiplatform") {
+            configuration("invalidJS") {
+                iterations = 1
+                iterationTime = 100
+                iterationTimeUnit = "ms"
+                reportFormat = "json"
+            }
+            register("jsTest")
+            kotlin {
+                js("jsTest", KotlinConfiguration.IR) {
+                    d8()
```

Comment:
Could you please also test when the `browser()` environment is selected?

Reply:
Done